src/fields/field.impl.hh

```diff
@@ -723,7 +725,10 @@ void Field<spacedim,Value>::fill_data_cache(OutputTime::DiscreteSpace space_type
 
             if (field_fe_ptr) {
                 auto native_output_data_base = stream->prepare_compute_data<double>(this->name(), space_type,
-                        (unsigned int)Value::NRows_, (unsigned int)Value::NCols_);
+                        (unsigned int)Value::NRows_, (unsigned int)Value::NCols_,
+                        typeid(field_fe_ptr->get_dofhandler()->ds()->fe()[0_d].get()).name(),  // should be used better solution of fe_type setting
+                                                                                               // e.g. method 'name()' of FiniteElement and descendants
```

Comment:
Use better setting of fe_type (eg. special method of FiniteElement and its descendants).
Add fe_type to output file and check correct outputs of appropriate tests.